pkg/assembler/backends/ent/artifact.go

```diff
@@ -0,0 +1,291 @@
+// Code generated by ent, DO NOT EDIT.
+
+package ent
+
+import (
+	"fmt"
+	"strings"
+
+	"entgo.io/ent"
+	"entgo.io/ent/dialect/sql"
+	"github.com/guacsec/guac/pkg/assembler/backends/ent/artifact"
+)
+
+// Artifact is the model entity for the Artifact schema.
+type Artifact struct {
```

Comment:
Question: When the GraphQL schema changes will this be autogenerated or we'll have to update?

Reply:
This schema will need to be manually edited and then run the code gen for ent.